### PR TITLE
8.0 update travis config

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by gurneyalex** [comment]
failure on runbot is "normal": there should be no dependency on aeroo reports in OCA addons.

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
+  - LINT_CHECK="1"
+  - TRANSIFEX="1"
+  - ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - ODOO_REPO="OCA/OCB" LINT_CHECK="0"
 
 virtualenv:
   system_site_packages: true
 
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```
Comment (tafaRU) [inline .travis.yml:37]: Please add `--depth=1`

**Comment by tafaRU** [comment]
@gurneyalex, thanks!
One minor issue. The rest LGTM :+1:

**Comment by lepistone** [comment]
Thanks for the review! We need to merge #63 first (a revert) can you please give a look @tafaRU ? Then I'll rebase this one.

**Comment by lepistone** [comment]
Superseded by #64.

**Pull request closed without merging**